app/videobridge/client/actionLink.js

```diff
@@ -15,7 +15,7 @@ actionLinks.register('joinJitsiCall', function(message, params, instance) {
 		const currentTime = new Date().getTime();
 		const jitsiTimeout = new Date((room && room.jitsiTimeout) || currentTime).getTime();
 
-		if (room && room.muted.includes(username)) {
+		if (room && (room.muted || []).includes(username)) {
```

Comment:
We could try a shortcut with nullish coalescing and optional chaining:

```suggestion
		if (room?.muted?.includes(username)) {
```

Reply:
Good suggestion 🙂 . I see we have some more places using `(room.muted || [])`. I'll replace them all by `room?.muted?.`

Reply:
Fixed!